mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
```

Comment:
Please move the content from `mypy.ini` into `setup.cfg`.
In the last file we have mypy config options already.
Let's be consistent.

Reply:
Makes sense, I just pushed that change.